plugins/rebalance/api/req_resp.go

```diff
@@ -0,0 +1,74 @@
+package rebalance
+
+import (
+	"github.com/pborman/uuid"
+)
+
+// Status represents Rebalance Status
+type Status uint64
+
+const (
+	// NotStarted should be set only for a node in which rebalance process is not started
+	NotStarted Status = iota
+	// Started should be set only for a node that has been just started rebalance process
+	Started
+	// Stopped should be set only for a node that has been just stopped rebalance process
+	Stopped
+	// Complete should be set only for a node that the rebalance process is completed
+	Complete
+	// Failed should be set only for a node that are failed to run rebalance process
+	Failed
+)
+
+// Command represents Rebalance Commands
+type Command uint64
+
+const (
+	// CmdNone should be set only when given cmd is none
+	CmdNone Command = iota
+	// CmdStart should be set only when given cmd is rebalance start
+	CmdStart
+	// CmdStop should be set only when given cmd is rebalance stop
+	CmdStop
+	// CmdStatus should be set only when given cmd is rebalance status
+	CmdStatus
+	// CmdFixlayoutStart should be set only when given cmd is rebalance fix-layout start
```

Comment:
update const name  in comment

Reply:
Can you clarify which name you mean?

Reply:
I guess the comment has small "l" in CmdFixlayoutStart, while the constant has a capital L

Reply:
Done